### ISSUE TITLE
Master Thesis Design

### ISSUE THREAD

**Issue by Christovis** [kntziora/MSc-SD-for-SET-in-NL#4]
`In order to answer to the research questions, the following supporting sub-questions are formulated`
Maybe rephrase as: `To facilitate the investigation, we populate intermediate goals underneath this overarching research question which we formulate as follows:`

`Therefore, much interest has been shown in energy transition from policy makers, governments and academics, due to its environmental efficiency [9]`
To me it sounds like 'the broader society is interested in energy transition due to its environmental efficiency', which seems odd (transition != the state of env. efficiency we aim for ?). Could you maybe reword this a bit.

`For instance, Keller, organises the models in three different types [1] depending on their functionalities.` If those three types can be listed in a concise manner, I would do so. Since it is mentioned that there are three types one would like to know what they are.

I will try to add more things later today.

**Comment by kntziora** [comment]
Thank you for your comments, I will the feedback incorporate them in the design. 

Also, another thing I would like to ask: I haven't mentioned the institutions that are involved in the research yet. Should I include this information or it is not necessary at the moment?

Thank you, have a great rest of the day!

**Comment by Christovis** [comment]
What institutions do you mean?
I am not sure about that. Maybe Vitor has an idea how to go about this.

**Comment by Christovis** [comment]
Simply adding UvA should be sufficient.

**Comment by vvvasconcelos** [comment]
You can add both IvI and IAS.
